src/routes/Router/RouterPath.tsx

```diff
@@ -0,0 +1,13 @@
+export enum RouterPath {
+  OfferedApiDelegations = 'offered-api-delegations',
+  Overview = 'overview',
+  ChooseApi = 'choose-api',
+  ChooseOrg = 'choose-org',
+  Confirmation = 'confirmation',
+  Receipt = 'receipt',
+  ReceivedApiDelegations = 'received-api-delegations',
+  Profile = 'Profile',
+  BasePath = '/accessmanagement/ui',
+  ProfileAT23 = 'https://at23.altinn.cloud/ui/Profile',
```

Comment:
How does the frontend/client know which environment it's running in to select either ProfileAT23 or ProfileProd (and all the other environments it will be deployed to)?

Reply:
Could it instead take it's own current domain: https://am.ui.{env}.altinn.cloud/
and just drop the am.ui subdomain?

Reply:
Not sure how that would play out. Profile, is on a completely different domain than the frontend since it's still only in SBL. It is dropping the am.ui subdomain right now.

Reply:
The domain for TT02 is: `https://tt02.altinn.no/ui/Profile` 
it will be the same across all environments that we'll need to just drop the am.ui subdomain. So if that's whats already is being done I guess the environment specific values here is not needed?

Otherwise the BFF will always know it's deployed environment through environment deploy variables, so the client could receive this info on first initialization perhaps

Reply:
Oh yeah, now I understand what you mean. I'll try it!